fetch.bs

```diff
@@ -4383,6 +4383,13 @@ steps:
 
      <li><p>Let <var>bodyInfo</var> be <var>response</var>'s <a for=response>body info</a>.
 
+     <li><p>Let <var>mimeType</var> be the empty string if <var>fetchParams</var>'s
+     <a for="fetch params">request</a>'s <a for=request>mode</a> is "<code>navigate</code>" and
+     <var>response</var>'s <a for=response>URL</a>'s <a for=url>origin</a> is not
+     <a>same origin</a> with <var>request</var>'s <a for=request>origin</a>, and
+     result of <a for="header list">extracting a MIME type</a> from <var>response</var>'s
```

Comment:
Can this be called with an opaque response?

Reply:
Yeah, when it's opaque we have to keep an empty string. 

This seems to effectively be the same condition that we are applying to `responseStatus` as well. Would it make sense to keep the result of this condition in a variable and use it to determine `responseStatus`, `contentType` and others fields if any that would have to be gated on CORS.

Something like : 
Let _shouldReport_ be `true` if response's type is  "opaque"  or  fetchParams ’s request ’s mode is "navigate " and  response’s url ’s origin is not same origin with  request’s origin, and `false` otherwise.

Let _mimeType_ be result of extracting a MIME type from response’s  header list if _shouldReport_ is true and 0 otherwise.

Let _responseStatus_ be response's status if _shouldReport_ is true and 0 otherwise.

Reply:
What @noamr did elsewhere (e.g. with "body info" is to only initialize it when the response is not opaque. It might be worthwhile to repeat the same pattern (or hook the info you want here into "body info" itself, as mime type kinda feels related.

@noamr - thoughts?

Reply:
> What @noamr did elsewhere (e.g. with "body info" is to only initialize it when the response is not opaque. It might be worthwhile to repeat the same pattern (or hook the info you want here into "body info" itself, as mime type kinda feels related.
> 
> @noamr - thoughts?

Yes mime-type should be inside `body info`.

Reply:
I've moved content-type inside `body info`. But can we rely on `body info`'s initialization for `content type` as well ? 

`body info` initialization seems to depend on [timing allow passed flag](https://fetch.spec.whatwg.org/#concept-response-timing-allow-passed). In [fetch response handover](https://fetch.spec.whatwg.org/#fetch-finale) 3.3.5, `body info` is reinitialized depending on `timing allow passed flag` which is different from the conditions we need for `content type`